libs/coin-modules/coin-cosmos/src/api/Cosmos.ts

```diff
@@ -231,6 +231,8 @@ export class CosmosAPI {
       BOND_STATUS_UNBONDED: "unbonded",
       BOND_STATUS_UNBONDING: "unbonding",
       BOND_STATUS_BONDED: "bonded",
+      // TO CHECK
```

Comment:
I think it's something I added right ? We need to check if this value can really happen

Reply:
It is indeed a possible value: https://docs.cosmos.network/api#tag/Query/operation/Validator 
<img width="612" alt="image" src="https://github.com/user-attachments/assets/de5b71d1-c883-4719-b0e4-db1d9388a098">

Reply:
Ok let's remove the comment and resolve the thread :+1:

Reply:
Done here 👍 https://github.com/LedgerHQ/ledger-live/pull/7636/commits/58a67659a8c847a2889c0d5ce654f27a798af8ba